### PR TITLE
Update environments-virtual-machines.md

### PR DESCRIPTION
**Pull request by marleixo**
Added permissions pre-requisites on correspondent deployment pool

### CODE REVIEW

**Comment by PRMerger20** [comment]
@marleixo : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ktoliver** [comment]
#label:"aq-pr-triaged"

**Comment by juliakm** [comment]
Thanks @marleixo! 
#sign-off

File: docs/pipelines/process/environments-virtual-machines.md [hunk 1]
```diff
@@ -26,6 +26,8 @@ You must have at least a Basic license and access to the following areas:
 
 For more information about security for Azure Pipelines, see [Pipeline security resources](../security/resources.md).
 
+To add a VM into an environment you must have Administrator permission at the correspondent Deployment pool
```
Comment (ShannonLeavitt) [inline docs/pipelines/process/environments-virtual-machines.md:29]:
```suggestion
To add a VM into an environment, you must have Administrator permission at the corresponding deployment pool.
```
Minor edits

**Pull request merged**